packages/legend-application-repl/src/stores/WindowNotificationManager.tsx

```diff
@@ -0,0 +1,78 @@
+import type { GenericLegendApplicationStore } from '@finos/legend-application';
+import {
+  DEFAULT_SMALL_ALERT_WINDOW_CONFIG,
+  LayoutConfiguration,
+  WindowState,
+  type LayoutManagerState,
+  type WindowConfiguration,
+} from './LayoutManagerState.js';
+import {
+  type AlertAction,
+  Alert,
+  AlertType,
+} from '../components/repl/Alert.js';
+
+export class WindowNotificationManager {
```

Comment:
No need to separate this out right now I think, let's leave it as part of `DatacubeStore` till we need to abstract this out and maybe move it to `application` layer